docs/section10/10-6.md

```diff
@@ -0,0 +1,34 @@
+# Airside systems – Rule 10-6
+**Schema Version:** 0.0.12  
+**Mandatory Rule:** False   
+**Rule ID:** 10-6
+ 
+**Rule Description:** For HVAC systems designed, mechanical cooling equipment efficiencies shall be adjusted to remove the supply fan energy from the efficiency rating.  
+**Rule Assertion:** P-RMR = expected value                                            
+**Appendix G Section:** Table G3.1 Section 10 b.  
+**Appendix G Section Reference:** None  
+**Data Lookup:** None  
+**Evaluation Context:** Evaluate each CoolingSystem data table  
+
+**Applicability Checks:** 
+
+1. HVAC with mechanical cooling.
+
+**Function Call:** None
+
+**Manual Check:** 
+
+ 1. Check that the mechanical equipment cooling efficiency has been adjusted to remove the supply fan energy from the efficiency rating. 
+
+## Rule Logic:  
+**Applicability Check 1:** 
+- For each hvac system in the P_RMR: `for hvac_p in P_RMR...HeatingVentilationAirConditioningSystem:`
+    - For each cooling system in the HVAC system: `for cooling_system_p in hvac_p:`
+        - Check that cooling system type does not equal None:: `if cooling_system_p.cooling_system_type != "None":`
+            - Get the full load cooling efficiency: `full_load_efficiency_p = cooling_system_p.full_load_efficiency`
```

Comment:
The RDS may just be checking that CoolingSystem.full_load_efficiency_no_fan is entered. Also need to check that HeatingSystem.full_load_efficiency_no_fan is provided. There is currently no such field in the RMD, so I added an issue on github

Reply:
@mkarpman  There are separate checks for heating and cooling. See 10-13

Reply:
Updated per your suggestion (please check if I got the outcomes correct). Need to update 10-13 as well.